### DISCUSSION TITLE
Vanilla Theme

### DISCUSSION THREAD

**Discussion by GamingKlex** [jarek-foksa/xel#109] [Q&A] [answered]
Will the vanilla theme ever come back?

**Comment by jarek-foksa** [comment, marked as answer]
The current plan is to make the toolkit look native on every platform, so instead of a generic Vanilla theme my goal is to introduce more native-like themes such as [Material](https://boxy-svg.com/ideas/258/material-theme) and [Spectrum](https://boxy-svg.com/ideas/257/spectrum-theme).
